vaadin-date-time-picker-flow-testbench/src/main/java/com/vaadin/flow/component/datetimepicker/testbench/DateTimePickerElement.java

```diff
@@ -87,7 +150,102 @@ protected void setValue(String value) {
      * @return the value of the date time picker or an empty string if no date
      *         time is selected
      */
-    protected String getValue() {
-        return getPropertyString("value");
+    private String getValue() {
+        return getPropertyString(VALUE_PROPERTY);
+    }
+
+    /**
+     * Sets the selected date of the inner DatePicker as a string.
+     * <p>
+     * The value is based on ISO 8601 e.g. <code>YYYY-MM-DD</code>.
+     *
+     * @param value
+     *            the value to set
+     */
+    private void setDateValue(String value) {
+        $(DATE_PICKER_TAG_NAME).attribute(SLOT_ATTRIBUTE, DATE_PICKER_SLOT_NAME)
+                .first().setProperty(VALUE_PROPERTY, value);
+        triggerCustomFieldUpdateValue();
+    }
+
+    /**
+     * Gets the selected date of the inner DatePicker as a string.
+     * <p>
+     * The value is based on ISO 8601 e.g. <code>YYYY-MM-DD</code>.
+     *
+     * @return the value of the inner date picker or an empty string if no date
+     *         is selected
+     */
+    private String getDateValue() {
+        return $(DATE_PICKER_TAG_NAME)
+                .attribute(SLOT_ATTRIBUTE, DATE_PICKER_SLOT_NAME).first()
+                .getPropertyString(VALUE_PROPERTY);
+    }
+
+    /**
+     * Sets the selected time of the inner TimePicker as a string.
+     * <p>
+     * The value is based on ISO 8601 e.g. <code>hh:mm:ss</code>.
+     *
+     * @param value
+     *            the value to set
+     */
+    private void setTimeValue(String value) {
+        $(TIME_PICKER_TAG_NAME).attribute(SLOT_ATTRIBUTE, TIME_PICKER_SLOT_NAME)
+                .first().setProperty(VALUE_PROPERTY, value);
+        triggerCustomFieldUpdateValue();
+    }
+
+    /**
+     * Gets the selected time of the inner TimePicker as a string.
+     * <p>
+     * The value is based on ISO 8601 e.g. <code>hh:mm:ss</code>.
+     *
+     * @return the value of the inner time picker or an empty string if no time
+     *         is selected
+     */
+    private String getTimeValue() {
+        return $(TIME_PICKER_TAG_NAME)
+                .attribute(SLOT_ATTRIBUTE, TIME_PICKER_SLOT_NAME).first()
+                .getPropertyString(VALUE_PROPERTY);
+    }
+
+    /**
+     * Trigger `__updateValue()` on custom field. This is needed when simulating
```

Comment:
Backticks don't work in JavaDocs, need to use either
`{@code foo}`
or
`<code>foo</code>`

Reply:
Updated.